scss/components/side-nav.scss

```diff
@@ -133,3 +136,7 @@ $block: #{$fd-namespace}-side-nav;
         }
     }
 }
+ul, ol {
+    margin-top: 0;
+    margin-bottom: 0;
+}
```

Comment:
Styles should not be tied to element selectors (`<ul>` and `<ol>` in this case) as styles can bleed in or out to other components on a page.